gnucash/register/register-gnome/combocell-gnome.c

```diff
@@ -501,6 +519,57 @@ gnc_combo_cell_set_value (ComboCell *cell, const char *str)
     gnc_basic_cell_set_value (&cell->cell, str);
 }
 
+/* This function looks through full_store for a partial match with newval and returns the first
+ match (which must be subsequently freed). It fills out box->item_list with found matches */
+static gchar* gnc_combo_cell_type_ahead_search(const gchar* newval, GtkListStore* full_store, PopBox *box)
```

Comment:
Decls get everything on one line, but impls should have the storage class and return type on one line with the function name on a second line. Aside from style consistency it makes it easy to grep for the implementation, e.g.
`grep -r ^gnc_combo_cell_type_ahead_search *gnucash`.

Reply:
ok I see, never thought of that.